tests/functional/adapter/test_python_submissions.py

```diff
@@ -1,89 +1,175 @@
-# import pytest
-# import yaml
```

Comment:
I'm not sure why the preexisting tests in this file were commented out given that they run correctly. I decided to uncomment them, but let me know if there's a reason to keep them disabled and I can disable everything but my new test.